style.css

```diff
@@ -0,0 +1,106 @@
+
+
+body {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--color-white);
+    margin: 0;
+  }
+  
+  body::before {
+    content: '';
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    z-index: -1;
+    background: var(--color-darkblue);
+    background-image: linear-gradient(
+        115deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+      ),
+      url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
```

Comment:
Please you code must be clear and clean. Work on your linters error.

Reply:
- [ ] Your comment is not clear, **what exactly is not clear & clean about the code block you have highlighted** 🤔? You need to point it out clearly so there is no need to ask the question above. Having linter errors does not mean your code is not clean.